repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/contactCard.ftl

```diff
@@ -0,0 +1,55 @@
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
```

Comment:
@thojava - Could we keep dash based naming convention for contactCard like `contact-card.ftl` in order to be inline with other macros ?

Reply:
yeap!